its/plugin/src/test/java/com/sonar/it/web/FileSuffixesTest.java

```diff
@@ -1,19 +1,18 @@
 /*
- * SonarSource :: HTML :: ITs :: Plugin
- * Copyright (c) 2011-2024 SonarSource SA and Matthijs Galesloot
- * sonarqube@googlegroups.com
+ * SonarQube HTML
```

Comment:
Why is the name 'SonarQube' and not 'SonarSource' ?

Reply:
I aligned to what is done in sonar-java: https://github.com/SonarSource/sonar-java/blob/master/sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaPlugin.java